src/components/transaction/transaction.vue

```diff
@@ -99,9 +87,6 @@ export default {
   },
   filters: { txTypeToName },
   computed: {
-    componentForTx () {
-      return txTypesComponentsMap[ this.transaction.tx.type ]
-    },
```

Comment:
This code duplicates in [`TransactionDetail` screen](https://github.com/aeternity/aepp-blockchain-explorer/blob/44de5fbde0ab18f68ff830373a5e3f16ff212b72/src/screens/transactionDetail/transactionDetail.vue#L134-L136). Let's sync changes between them until it is extracted.